apps/web/playwright/login.2fa.e2e.ts

```diff
@@ -169,6 +169,7 @@ async function fillOtp({ page, secret, noRetry }: { page: Page; secret: string;
     // Maybe token was just about to expire, try again just once more
     token = authenticator.generate(secret);
   }
+  await page.locator('input[name="2fa1"]').waitFor({ state: "visible", timeout: 10_000 });
```

Comment:
Not sure why this test sometimes fails on the line 

`await page.fill('input[name="2fa1"]', token[0]);`.

added waitFor with timeout